doc/api/stream.md

```diff
@@ -432,6 +432,17 @@ file.end('world!');
 // Writing more now is not allowed!
 ```
 
+[`stream.end()`][stream-end] will error with (in order of precedence):
+* `ERR_STREAM_WRITE_AFTER_END` if `chunk` is not nully and
+[`stream.end()`][stream-end] has been called.
+* `ERR_STREAM_DESTROYED` if `chunk` is not nully and
+[`stream.destroy()`][stream-destroy] has been called.
+* `ERR_INVALID_ARG_TYPE`  if `chunk` is not nully and
+`chunk` is not a `string` when not in `objectMode`.
+* `ERR_STREAM_ALREADY_FINISHED` if
+[`writable.writableFinished`][].
+* Any error emitted by the instance through `'error'`.
```

Comment:
This one is not strictly true in the current master, but I believe that's a bug.